docs/python/testing.md

```diff
@@ -197,34 +191,19 @@ You run tests using any of the following actions:
 
       ![Running tests at specific scopes through Test Explorer](images/testing/test-explorer-run-scoped-tests.png)
 
-- Right-click a file in Explorer and select **Run All Tests**, which runs the tests in that one file.
+  - You can also run a selection of tests through the test explorer. To do that,  `kbstyle(Ctrl+Click)` (or `kbstyle(Cmd+Click)` on macOS) on the tests you wish to run, right-click on one of them and then select **Run Test**.
 
-- From the **Command Palette**, select any of the run test commands:
 
-    ![Python testing commands on the Command Palette](images/testing/commands.png)
+After a test run, VS Code displays results directly in the editor as gutter decorations. Failed tests will also be highlighted in the editor, with a Peek View that displays the test run error message along with a history of all of the tests' runs. You can press `escape` to dismiss the view, and you can disable it by opening the User settings (**Preferences: Open Settings (UI)** command in the **Command Palette**) and changing the value of the `Testing: Automatically Open Peek View` setting to `never`.
 
-    | Command | Description |
-    | --- | --- |
-    | Debug All Tests | See [Debug tests](#debug-tests). |
-    | Debug Test Method | See [Debug tests](#debug-tests). |
-    | Run All Tests | Searches for and runs all tests in the workspace and its subfolders. |
-    | Run Current Test File | Runs the test in the file that's currently viewed in the editor. |
-    | Run Failed Tests | Re-runs any tests that failed in a previous test run. Runs all test if no tests have been run yet. |
-    | Run Test File | Prompts for a specific test filename, then runs the test in that file. |
-    | Run Test Method | Prompts for the name of a test to run, providing auto-completion for test names. |
-    | Show Test Output | Opens the Python Test Log panel with information about passing and failing tests, as well as errors and skipped tests. |
-
-After a test run, VS Code displays results directly with the CodeLens adornments in the editor and in **Test Explorer**. Results are shown both for individual tests as well as any classes and files containing those tests. Failed tests are also adorned in the editor with a red underline.
+In the **Test Explorer**, results are shown for individual tests as well as any classes and files containing those tests.
 
 ![Test results on a unittest class and in Test Explorer](images/testing/test-results.png)
 
-VS Code also shows test results in the **Python Test Log** output panel (use the **View** > **Output** menu command to show the **Output** panel, then select **Python Test Log** from the dropdown on the right side):
+VS Code also shows test results in the **Python Test Log** output panel.
```

Comment:
Should we mention the button in the test explorer to bring up the UI?.

Reply:
I believe mentioned it on the configure tests section 😊